Test/XLTestNGlyco.cs

```diff
@@ -25,7 +25,8 @@ public class XLTestNGlyco
         [Test]
         public static void GlyTest_GetKindString()
         {
-            byte[] kind = new byte[] {3, 4, 0, 0, 1, 0, 0, 0, 0, 0 };
+            byte[] kind = new byte[Glycan.SugarLength];
```

Comment:
here is where you can test file reading and making sure that errors in the file edited by the user are handled without crash.